src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -328,6 +341,12 @@ private function getListenerInfo($listener, $eventName)
                 $listener = array($listener, '__invoke');
             }
             $class = is_object($listener[0]) ? get_class($listener[0]) : $listener[0];
+
+            $listenerHash = md5($class);
```

Comment:
@xabbuh yes! But I'd like more people to try this and see if it improves for them. I'd also like some confirmation about `ProfilerListener` being super slow in real applications. Thanks!